agreement/player.go

```diff
@@ -421,14 +421,23 @@ func (p *player) partitionPolicy(r routerHandle) (actions []action) {
 		bundlePeriod = b.Period
```

Comment:
This PR seems to make the comment a few lines above this no longer true, right? (lines 402 through 414)